doc/extensions/local_util.py

```diff
@@ -0,0 +1,57 @@
+# Utility functions used in conf.py
+#
+# Copyright 2017 Espressif Systems (Shanghai) PTE LTD
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import unicode_literals
+from io import open
+import os
+import shutil
+
+
+def run_cmd_get_output(cmd):
+    return os.popen(cmd).read().strip()
```

Comment:
I think if you use subprocess.popen (https://docs.python.org/3/library/subprocess.html#popen-constructor) you can direct stderr to /dev/null so the command is quieter for our cases.